ln/resubscribe_invoices.ts

```diff
@@ -29,9 +31,9 @@ const resubscribeInvoices = async bot => {
     }
     logger.info(`Invoices resubscribed: ${invoicesReSubscribed}`);
   } catch (error) {
-    logger.error(`ResubcribeInvoice catch: ${error.toString()}`);
+    logger.error(`ResuscribeInvoice catch: ${String(error)}`);
```

Comment:
@Mersho typo here, don't remove the 'b'

Reply:
@Mersho actually, there was a typo before too, the correct way to fix it is not replacing the `b` with an `s` but just adding the extra `s` without removing the `b`

Reply:
Good catch.